src/zcash/Address.hpp

```diff
@@ -163,22 +173,26 @@ class UnifiedFullViewingKeyBuilder;
 /**
  * Wrapper for a zcash_address::unified::Ufvk.
  */
-class UnifiedFullViewingKey {
+class UnifiedFullViewingKey
+{
 private:
     std::unique_ptr<UnifiedFullViewingKeyPtr, decltype(&unified_full_viewing_key_free)> inner;
 
-    UnifiedFullViewingKey() :
-        inner(nullptr, unified_full_viewing_key_free) {}
+    UnifiedFullViewingKey() : inner(nullptr, unified_full_viewing_key_free) {}
 
-    UnifiedFullViewingKey(UnifiedFullViewingKeyPtr* ptr) :
-        inner(ptr, unified_full_viewing_key_free) {}
+    UnifiedFullViewingKey(UnifiedFullViewingKeyPtr* ptr) : inner(ptr, unified_full_viewing_key_free)
+    {
+    }
 
     friend class UnifiedFullViewingKeyBuilder;
+
 public:
     UnifiedFullViewingKey(UnifiedFullViewingKey&& key) : inner(std::move(key.inner)) {}
 
-    UnifiedFullViewingKey(const UnifiedFullViewingKey& key) :
-        inner(unified_full_viewing_key_clone(key.inner.get()), unified_full_viewing_key_free) {}
+    UnifiedFullViewingKey(const UnifiedFullViewingKey& key)
+        : inner(unified_full_viewing_key_clone(key.inner.get()), unified_full_viewing_key_free)
+    {
+    }
```

Comment:
If we set [SplitEmptyFunction: false](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#bracewrapping) as suggested [here](https://github.com/zcash/zcash/pull/6678/files#r1200693772), then I think this becomes
```c++
    UnifiedFullViewingKey(const UnifiedFullViewingKey& key)
        : inner(unified_full_viewing_key_clone(key.inner.get()), unified_full_viewing_key_free)
    {}
```
which is less verbose, while still avoiding the issue that the `{}` is easily missed at the end of the line.